report/src/main/webapp/app/components/keyboardShortcuts.directive.js

```diff
@@ -86,6 +90,20 @@ define(['angularAMD', 'userSettingsService'], function (angularAMD) {
         }
       };
 
+      function clickButton(buttonClass, event) {
+        var $tab = (event.shiftKey) ? $('.tab-content-toolbar') : $('.toolbar-bottom');
+        var $btns = $tab.find('.toolbar-btns');
+        $btns.find(buttonClass).click();
+      }
+
+      function accept(event) {
+        clickButton('.button-blue', event);
```

Comment:
I'm not sure that this is correct assumption, that element with class `button-blue` should be clicked on the accept event (the same for the `button-red`). More buttons also have those classes, e.g.: 
![2018-09-20_09h46_37](https://user-images.githubusercontent.com/19705838/45803511-14944380-bcba-11e8-90f9-8b35adfe3219.png)

Maybe buttons for accepting/reverting should have their own identifiers?

Reply:
Assumption is that only buttons inside "toolbar-bottom" or "tab-content-toolbar" will be clicked (function clickButton is looking for those classes first and then looking in their child nodes proper buttons for click). "Save all changes" button is inside "sidepanel" so it will not be clicked.